### DISCUSSION TITLE
VW header files

### DISCUSSION THREAD

**Discussion by rdurnin** [NeoGeographyToolkit/StereoPipeline#359] [Q&A] [answered]
Hello, and I am looking for the VW (Core.h, Image.h, FileIO.h and Cartography.h) header files for compiling some old code, and have been unable to locate them in the nightly or public release bundles. Can they be shared or will they be generated by compiling the code manually?

**Comment by oleg-alexandrov** [comment, marked as answer]
Those headers are in VisionWorkbench. The code is public, we just don't ship it with the compiled programs. See here: https://github.com/visionworkbench/visionworkbench/tree/master/src/vw.

How to build VisionWorkbench and StereoPipeline from source, while fetching the rest of the dependencies, is described here: https://stereopipeline.readthedocs.io/en/latest/installation.html#building-asp-from-source

Note that this is an advanced topic, and some work and knowledge of building code is expected.
